skimage/morphology/convex_hull.py

```diff
@@ -18,6 +18,32 @@ def _offsets_diamond(ndim):
     return offsets
 
 
+def _check_coords_in_hull(gridcoords, hull_equations, tolerance):
```

Comment:
It would be great to have a docstring here that briefly explains why this is needed (it's a memory-hungry method) and what it does. It can include a link to the issue, for example.

Reply:
Done